packages/blade/src/components/FileUpload/types.ts

```diff
@@ -70,9 +70,13 @@ type FileUploadCommonProps = {
    */
   maxSize?: number;
   /**
-   * Callback function triggered when files are selected
+   * Callback function triggered when files are selected or when a file is dropped into the upload area
    */
   onChange?: ({ name, fileList }: { name?: string; fileList: BladeFileList }) => void;
+  /**
+   * Callback function triggered when files are selected or when a file is dropped into the upload area
+   */
+  onInput?: ({ name, fileList }: { name?: string; fileList: BladeFileList }) => void;
```

Comment:
This seems same as `onChange `, are we planning to deprecate the `onChange` prop in favor of `onInput`? If so, we need to update the description accordingly.

Reply:
in case of <input type="file" /> onChange and onInput have same behaviour.
https://codepen.io/gauravTRazorpay/pen/wvVxMPM  but currently fileUpload don't have onInput prop. so FileUpload component does not fire  onInput event when the file is dropped.  we need to trigger native input events in file upload.